packages/web/src/pages/Honeypots/VaultDetailsPage/Sections/VaultSubmissionsSection/PublicSubmissionCard/PublicSubmissionCard.tsx

```
@@ -1,45 +1,57 @@
   return (
     <StyledPublicSubmissionCard isOpen={isOpen}>
-      <div className="card-header" onClick={() => setIsOpen((prev) => !prev)}>
-        <div className="severity">
-          <Pill textColor={severityColors[severityIndex ?? 0]} isSeverity text={submission.severity ?? t("noSeverity")} />
+      <div className="card-header">
+        <div onClick={() => setIsOpen((prev) => !prev)}>
+          {submission && submission?.validLabels.length > 0 && (
+            <div className="labels">
+              <span>{t("labeledAs")}:</span>
+              {submission.validLabels.map((label) => (
+                <div className="label">
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)